src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div class="select-one-response-block-contact-property-editor">
+    <generic-contact-property-editor
+      :block="block"
+      @toggleSetContactProperty="onSetContactPropertyToggle"
+      @changeContactPropertyType="resetMapping"
+      @updateShouldUseCurrentBlockResponse="onShouldUseCurrentBlockResponseUpdate">
+      <template slot="entry-from-this-block">
+        <div class="mt-2 mb-2">
+          <div
+            v-if="choiceKeys.length === 0"
+            class="mt-2 mb-2 text-danger">
+            {{ trans('flow-builder.enter-at-least-one-choice-above') }}
```

Comment:
Would this be visible as validation errors, which would block flow PUBLISH action?

Reply:
I've checked now - no, it's not in the json schema for the `ISelectOneResponseBlockConfig`

Reply:
as discussed, we will leave this. And in addition I added it in https://github.com/FLOIP/flow-runner/pull/87